boards/arm/frdm_k64f/pinmux.c

```diff
@@ -155,6 +155,19 @@ static int frdm_k64f_pinmux_init(const struct device *dev)
 		       PORT_PCR_PE_MASK | PORT_PCR_PS_MASK);
 #endif
 
+#if CONFIG_SHIELD_WAVESHARE_EPAPER_GDEH029A1 ||   \
```

Comment:
I think that it is not what we should do. This can quickly become very confusing. I would like to know @MaureenHelm  opinion.

@KimBP You could move this commit, along with the similar commit from #30960, to a separate PR so it would not block here.